ipaclient/install/client.py

```diff
@@ -663,20 +663,24 @@ def hardcode_ldap_server(cli_server):
         "hardcoded server name: %s", cli_server[0])
 
 
-def configure_krb5_conf(
-        cli_realm, cli_domain, cli_server, cli_kdc, dnsok,
-        filename, client_domain, client_hostname, force=False,
-        configure_sssd=True):
-
-    # First, write a snippet to krb5.conf.d.  Currently this doesn't support
-    # templating, but that could be changed in the future.
+# Currently this doesn't support templating, but that could be changed in the
+# future.  Note that this function is also called from %post.
+def configure_krb5_snippet():
```

Comment:
I think we should also add `tasks.restore_context()` for the file added here. I found out that SELinux policy doesn't explicitly label `/etc/krb5.conf.d/*` other than `etc_t` but this will change in the future.